FueledUtils/UIExtensions.swift

```diff
@@ -127,4 +153,10 @@ public extension UIImage {
 		}
 		return image.resizableImage(withCapInsets: self.capInsets, resizingMode: self.resizingMode).withRenderingMode(self.renderingMode)
 	}
+
+	public func resized(offset: CGPoint = .zero, size: CGSize) -> UIImage? {
```

Comment:
@stephane-fueled when you display a large image in a small image view, the render server resizes the image internally (I think it's a hardware resize using linear interpolation) which is fast because the image stays within the renderer context. Using the method above will cause the image to be copied between processes which will generally slow down performance. We don't want to do it on the main thread during scrolling for example. On the other hand, using too large images will cause constant resizing in the renderer which may also slow down the app on screen. So there's choice to be made between resizing once and caching and built-int resizing (which is fine most of the time when the source image size is adequate).